scripts/eval/yamls/mpt_eval.yaml

```diff
@@ -2,31 +2,37 @@ max_seq_len: 1024
 tokenizer_name: EleutherAI/gpt-neox-20b
 seed: 1
 precision: amp_fp16
-
+run_name: mpt-eval
 
 models:
 -
   model_name: mpt_test
+
   # Tokenizer
   tokenizer:
     name: ${tokenizer_name}
     kwargs:
       model_max_length: ${max_seq_len}
   model:
     name: mpt_causal_lm
-    init_device: meta
-    d_model: 768
-    n_heads: 12
-    n_layers: 12
+    init_device: mixed
```

Comment:
imo the values here should match the _default_ values from configuration_mpt, even though you had to change things to have this yaml run against the test checkpoint. Because this yaml is just an example. but maybe leave a note saying that the model config section must match the checkpoint you are loading in exactly

Reply:
@dakinggg Makes sense. All config values have defaults, do you think we should explicitly state all of them or just the ones that existed prior?

Reply:
also init_device by default is 'cpu', but will use 'mixed' for all eval yamls

Reply:
I think leave out the ones that match the defaults, just to reduce verbosity here.

Reply:
also stuff should fail if we change the defaults, cause we should not do that :)